tests/capture/test_make_plxpr.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for the make_plxpr function for capturing qnodes as jaxpr.
+"""
+
+from unittest.mock import call
+
+import numpy as np
+import pytest
+
+import pennylane as qml
+
+pytestmark = pytest.mark.jax
+
+jax = pytest.importorskip("jax")
+
+# must be below jax importorskip
+from pennylane.capture import make_plxpr  # pylint: disable=wrong-import-position
+
+
+@pytest.fixture(autouse=True)
+def enable_disable_plxpr():
+    qml.capture.enable()
+    yield
+    qml.capture.disable()
+
+
+def test_make_plxpr(mocker):
+    """Test that make_plxpr uses make_jaxpr, and returns a callable that will
+    create a jaxpr representation of the qnode"""
+
+    dev = qml.device("default.qubit", wires=1)
+
+    spy = mocker.spy(jax, "make_jaxpr")
+
+    @qml.qnode(dev)
+    def circ(x):
+        qml.RX(x, 0)
+        qml.Hadamard(0)
+        return qml.expval(qml.X(0))
+
+    plxpr = make_plxpr(circ)(1.2)
+
+    spy.assert_called()
+    assert hasattr(plxpr, "jaxpr")
```

Comment:
Maybe more concretely could check the type itself?
```suggestion
    assert hasattr(plxpr, "jaxpr")
    isinstance(plxpr, jax._src.core.ClosedJaxpr)
```